code-of-conduct.md

```diff
@@ -21,131 +18,93 @@ Towards this end, certain behaviors and practices will not be tolerated.
 
 (We will add a physical mailing address here when we acquire one).
 
-Please make sure your notice meets the Digital Millennium Copyright Act
-requirements.
+Please make sure your notice meets the Digital Millennium Copyright Act requirements.
 
 ## Consequences
 
-All content published to public IPFS infrastructure is hosted at the sole
-discretion of the IPFS team.
+All content published to public IPFS infrastructure is hosted at the sole discretion of the IPFS team.
 
 Unacceptable behavior from any community member will not be tolerated.
 
 Anyone asked to stop unacceptable behavior is expected to comply immediately.
 
-If a community member engages in unacceptable behavior, the ipfs team
-may take any action they deem appropriate, up to and including a temporary ban
-or permanent expulsion from the community without warning (and without refund
-in the case of a paid event or service).
+If a community member engages in unacceptable behavior, the IPFS team may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event or service).
 
 ## Addressing Grievances
 
-If you feel you have been falsely or unfairly accused of violating this Code
-of Conduct, you should notify the IPFS team. We will do our best to ensure
-that your grievance is handled appropriately.
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify the IPFS team. We will do our best to ensure that your grievance is handled appropriately.
 
-In general, we will choose the course of action that we judge as being most in
-the interest of fostering a safe and friendly community.
+In general, we will choose the course of action that we judge as being most in the interest of fostering a safe and friendly community.
 
-On IRC, let one of the ops know if you think that someone has transgressed
-against the Code of Conduct. If you would like to be an op and promise to
-help maintain and abide by the code, please let us know.
+On IRC, let one of the ops know if you think that someone has transgressed against the Code of Conduct. If you would like to be an op and promise to help maintain and abide by the code, please let us know.
 
 ## Contact Info
 
-Please contact abuse@ipfs.io if you need to report a problem or address a
-grievance related to an abuse report.
+Please contact abuse@ipfs.io if you need to report a problem or address a grievance related to an abuse report.
+
+If you prefer, you can contact any of the abuse team members separately. The abuse team is formed by:
+
+- molly@ipfs.io
+- dietrich@ipfs.io
+- michael@ipfs.io
+- ebony@ipfs.io
+- hector@ipfs.io
+
+You are also encouraged to contact us if you are curious about something that might be "on the line" between appropriate and inappropriate content. We are happy to provide guidance to help you be a successful part of our community.
```

Comment:
Can you please add a statement about how reports are handled? We should be very clear about whether they're confidential or not, and whether reports shared with individual will be shared with other members of the group.

Reply:
Please see last changes and lmk if it looks good.